website/content/toc.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: '[TMP] Armada Documentation - Table of Contents'
+---
+
+> [!NOTE]
+> This page outlines a proposed structure for Armada's documentation website, designed to support three primary user journeys from initial discovery through advanced usage and contribution.
+>
+> **Purpose**: Establish a coherent structure and content framework to guide documentation development and ensure comprehensive coverage of user needs across different experience levels.
+>
+> **Target Audiences:**
+>
+> - **Users**: Data scientists, engineers, and analysts who submit and manage computational jobs
+> - **Operators**: Infrastructure teams responsible for deploying, scaling, and maintaining Armada clusters
+> - **Developers**: Developers building integrations, extending core functionality, or contributing to the open-source project
+>
+> **Next Steps**: This structure will evolve based on community feedback and real-world usage patterns. Maintainers and contributors are encouraged to share insights on content gaps, user pain points, and structural improvements.
+
+# 1. Overview
+
+**Introduction to Armada, its purpose, high-level architecture, and key components.**
+
+- What is Armada?
+- Why Use Armada?
+- Key Features and Benefits
+- Use Cases and Success Stories
+- Comparison with Other Schedulers
+
+---
+
+# 2. Getting Started
+
+**Start here if you're new to Armada. Learn how to install it and run your first job.**
+
+## 2.1 Quickstart
+
+- Prerequisites and Requirements
+- Quickstart with Kind & Helm
+- Submitting Your First Job
+- Viewing Job Status and Logs
+
+## 2.2 Tutorials
+
+- End-to-end ML Job Example
+- Batch Data Processing Pipeline
+- CI/CD Integration Walkthrough
+
+---
+
+# 3. Understanding Armada
+
+## 3.1 Architecture
+
+**Understand how Armada works under the hood.**
+
+- Armada system components
+- Scheduling pipeline and job lifecycle
+- Executors, queues, and priorities
+- Fault tolerance and scaling
+
+## 3.2 Core Concepts
+
+**Deep dives into key ideas and mechanisms that power Armada.**
+
+- Jobs & Queues
+- Prioritization and Fair-Use Scheduling
+- Scheduling Algorithms
+
+---
+
+# 4. Operator Guide
+
+**Complete guide for platform engineers and operators - from installation to production maintenance.**
+
+- Local installation (Kind, Minikube)
+- Installation and Deployment
+- Cluster setup and configuration
+- Monitoring and observability
+- Scaling
+- Troubleshooting
+
+---
+
+# 5. User Guide
+
+**Practical guidance for job submission and management.**
+
+- Submitting and cancelling jobs
+- Working with job specs
+- Using Integrations
+- Examples
+
+## 5.1 CLI Reference
+
+- Installing `armadactl`
+- CLI commands and flags
+- Example usage patterns
+
+## 5.2 API Reference
+
+- REST and gRPC APIs
+- OpenAPI Spec (auto-generated)
+
+## 5.3 Clients
+
+**Client libraries and SDKs for different languages.**
+
+- Python
+- Java/Scala
```

Comment:
Go and .NET

Reply:
Added, but I am unable to find the Go client, could you please provide a reference ?